packages/lesswrong/components/ea-forum/users/EAUsersProfile.tsx

```diff
@@ -110,6 +110,7 @@ const styles = (theme: ThemeType): JssStyles => ({
     marginBottom: 8,
     fontFamily: theme.palette.fonts.sansSerifStack,
     fontWeight: 500,
+    overflowWrap: "anywhere",
```

Comment:
I think `break-word` would be slightly better here 🤷‍♀️

Reply:
I don't think it really makes any difference here. The only difference is that "with `break-word`, the browser doesn’t consider the soft wrap opportunities introduced by the word break when calculating `min-content` intrinsic sizes, but it does with `anywhere`". It turns out that [break-word](https://caniuse.com/?search=overflow-wrap%20break-word) does have much better browser support than [anywhere](https://caniuse.com/?search=overflow-wrap%20anywhere) though, so it's probably worth changing for that reason.

```suggestion
    overflowWrap: "break-word",
```